model/project_aliases.go

```diff
@@ -71,38 +71,110 @@ type ProjectAlias struct {
 type ProjectAliases []ProjectAlias
 
 // FindAliasesForProject fetches all aliases for a given project
```

Comment:
Should update this comment for the version string

Reply:
Actually I haven't found a place where we're using the version string. Do we need to input this or should we always use the empty version?